src/NuGet.Core/NuGet.Protocol/Model/AlternatePackageMetadata.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Versioning;
+
+namespace NuGet.Protocol
+{
+    public class AlternatePackageMetadata
```

Comment:
I tried to keep the structure of alternate package metadata aligned with the structure of dependencies. Can we reuse the same class that represents dependency info here instead of adding this class?

Reply:
`PackageDependency` contains `Include` and `Exclude` properties which are not needed for this deprecation metadata.